### PR TITLE
Bump the handlebars helpers

### PR DESCRIPTION
**Pull request by LaurentGoderre**
Fixes assemble/grunt-assemble#31

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
@jonschlinkert I am using the 0.6.0 handlebars-helpers combination for which the helpers have been tested against.

**Comment by jonschlinkert** [comment]
@LaurentGoderre how extensive are the tests? I'm not sure how compatible the 0.6.0 helpers are.

**Comment by LaurentGoderre** [comment]
@jonschlinkert I just updated the tests. 0.6.0 was not quite compatible but I think I fixed it. Can you review the change to this as well as the tests I wrote?
https://github.com/assemble/assemble-handlebars/blob/master/test/core_test.js

**Comment by LaurentGoderre** [comment]
@jonschlinkert I also ran the these with this in grunt-assemble and it all passed.

**Comment by LaurentGoderre** [comment]
@jonschlinkert any objection with me merging this?

**Comment by doowb** [comment]
@LaurentGoderre I'm testing this out by running it through a few projects. I think the reason that we haven't updated this is because when `handlebars-helpers@0.6.0` was published there were a lot of changes to how the helpers were registered and helpers were removed and added. Because of this, we'll need to bump the minor here and in `grunt-assemble` since these are breaking changes.

Have you tried the latest version of `handlebars-helpers` yet?

**Comment by LaurentGoderre** [comment]
I haven't tested the latest but I know that the version here fixes the issues mentioned at the top. That is what I'm trying to solve. I was thinking of bumping it and bumping minor version and do the same to assemble so users can choose which version they want.

File: index.js [hunk 1]
```diff
@@ -7,22 +7,24 @@
...
 
 // register built-in helpers
 exports.init = function (options, params) {
-  if (helpers && helpers.register) {
-    helpers.register(handlebars, options, params);
+  options = options || {};
+  if (options.handlebars) {
+    handlebars = options.handlebars;
   }
+  helpers(null, options);
```
Comment (doowb) [inline index.js]: this should just be `helpers(options)`, otherwise the `options` aren't used in this version of `handlebars-helpers`

**Comment by doowb** [comment]
We'll definitely have to bump the minors. Most of the helpers are working fine but upgrading a project will require template changes.

I commented in the code on a change that should be made, then I think this is okay to merge. We'll then need to update `grunt-assemble` and document some of the changes.

File: index.js (hunk 1 above)
Comment (LaurentGoderre) [inline index.js]: Done!

**Comment by LaurentGoderre** [comment]
Any update on this?

**Comment by doowb** [comment]
Sorry for the delay. I'll get this published and update `grunt-assemble` too.

**Pull request merged**